dvc/dependency/repo.py

```diff
@@ -85,6 +85,10 @@ def update(self, rev=None):
         with self._make_repo(locked=False) as repo:
             self.def_repo[self.PARAM_REV_LOCK] = repo.get_rev()
 
+    def get_outs(self, rev=None):
+        with self._make_repo(locked=False, rev=rev) as repo:
+            return repo.find_outs_by_path(self.def_path)
+
```

Comment:
This wont work if you have any git files that you are importing. This assumption is very fragile. We should probably stick with the way we did it before and the way we do it in import-url, where we don't set md5.